src/main/java/org/jabref/gui/preferences/PreferencesDialog.css

```diff
@@ -11,6 +11,15 @@
     -fx-padding: 0em 1em 0em 3em;
 }
 
+.preferencePaneContainer .scroll-bar {
```

Comment:
This fixes the problem for the preferences dialog, but the problem also appears in some other dialogs. Can you please generalize your solution (does it work without the `.preferencePaneContainer` selector?) and move it to the base.css. Thanks!

Reply:
Can you look at my solution? Is it enough generalized?